src/docs/product/profiling/index.mdx

```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
 
 </Note>
 
-Profiling allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments. It's the next-generation performance monitoring feature for your apps, removing the limitations of manually created tests, by looking at real user experiences. With profiling, you're building in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one.
+Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise, code-level insight into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```

Comment:
> manual test instrumentation

This sounds weird to me. What does manual test instrumentation mean here? @JonasBa can you confirm this is the best way to talk about this?